src/ui/component/common/form-components/form-field.jsx

```diff
@@ -141,10 +146,15 @@ export class FormField extends React.PureComponent<Props> {
           </div>
         );
       } else if (type === 'textarea') {
+        const hasCharCount = charCount !== undefined && charCount >= 0;
+        const countInfo = hasCharCount && (
+          <span className="comment__char-count">{`${charCount || ''}/${textAreaMaxLength}`}</span>
```

Comment:
Instead of an empty string, this should just be `0`

Reply:
Oh sure, missed that one.